plugins/ern_v0.14.0+/react-native-fast-image_v5.3.0+/config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "com.android.support:support-v4:27.1.1",
+      "com.github.bumptech.glide:glide:4.7.1",
+      "com.github.bumptech.glide:annotations:4.7.1",
+      "annotationProcessor:com.github.bumptech.glide:compiler:4.7.1",
```

Comment:
we may have to rethink `annotationProcessor`
https://github.com/DylanVann/react-native-fast-image/blob/master/android/build.gradle#L63

Reply:
Should be ok, it's supported thanks to a previous PR done by @tianling536

Reply:
I see - https://github.com/electrode-io/electrode-native/commit/67fb1c74c6d64f56c8e3b3b08658a96797a55e5d